tools/style.sh

```diff
@@ -16,6 +16,14 @@ echo -e "\n::group:: ===> check flake8..."
 flake8 onnx tools workflow_scripts
 echo -e "::endgroup::"
 
+echo -e "\n::group:: ===> check isort..."
+isort . --color --diff --check
+echo -e "::endgroup::"
+
+echo -e "\n::group:: ===> check black format..."
+black . --color --diff --check
+echo -e "::endgroup::"
```

Comment:
So after this PR, isort and black check will be "required" in ONNX CIs. I am good with this and I just want to mention that we need to reflect this in the document. I will take care of this by https://github.com/onnx/onnx/pull/4332.